src/eureka/S3_data_reduction/background.py

```diff
@@ -152,7 +152,21 @@ def writeBG_WFC3(arg):
     data['flux'] -= data.bg
     if hasattr(data, 'medflux'):
         data['medflux'] -= np.median(data.bg, axis=0)
-
+        
+    # Save BG value at source position and BG stddev (no outlier rejection)
+    data['skylev'] = (['time', 'x'], 
+                      np.zeros([data.flux.shape[0], data.flux.shape[2]]))
```

Comment:
This declaration is going to be problematic for NIRISS because it has an extra `order` dimension.  Instead, do the following...
`coords = list(data.coords.keys())`
`coords.remove('y')`
`data['skylev'] = (coords, np.zeros_like(data.flux[:, 0]))`

Then apply similar code for `skyerr` below.
`data['skyerr'] = (coords, np.zeros_like(data.skylev))`